vault-change-publisher/scripts/prepare-publication-evidence.py

```
@@ -72,12 +76,141 @@ def relative_path(root: str, absolute: str) -> str:
+            "GIT_OPTIONAL_LOCKS": "0",
+            "GIT_CONFIG_COUNT": "1",
+            "GIT_CONFIG_KEY_0": "core.fsmonitor",
+            "GIT_CONFIG_VALUE_0": "false",
+        }
+    )
+    return environment
+
+
+def git_bytes(repo: str, git_dir: str, *arguments: str) -> bytes:
+    """Read bounded local Git object data without filters or replacement refs."""
+    process = subprocess.Popen(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)